models/devicesim-1.0.x/api/generated.go

```diff
@@ -7230,2233 +7230,2233 @@ var (
 		0x0b, 0x96, 0x1e, 0xea, 0xab, 0x5c, 0x7c, 0x3a, 0x3d, 0x3f, 0xb9, 0x38, 0x7f, 0x7f, 0xf6, 0xeb,
```

Comment:
you can omit this file from the patch - it's only changing because there is a non deterministic way of calculating this  serialisation of the YANG schema. This will be fixed with https://github.com/openconfig/goyang/pull/235 

Also remove the `sdn-fabric` generated.go